socks/socks.go

```diff
@@ -162,30 +159,33 @@ func ParseAddr(s string) Addr {
 }
 
 // Handshake fast-tracks SOCKS initialization to get target address to connect.
+// Read RFC 1928 for request and reply structure and sizes.
```

Comment:
This line of comment is implementation detail. It should NOT be at the function level because it'll be exposed by Godoc.